### ISSUE TITLE
How to compile Pantry Party for iOS?

### ISSUE THREAD

**Issue by ualex73** [PantryParty/pantry_party#124]
Is there a guide/steps how to compile Pantry Party for iOS? I tried something with Xcode, but it didn't load it.

**Comment by arjes** [comment]
PantyParty is a nativescript app. They have some getting started guides
that are far better than anything I could write.

If you can't find one, or get stuck, let me know.

**Comment by ualex73** [comment]
Thanks, and i found the website https://docs.nativescript.org/ . When i have some more time, i will execute the steps and see how i far i get.

Maybe this is useful information to put in the README file on github?

**Comment by ualex73** [comment]
Finally got it compiled, but i needed to use node v14 (v16 does not compile with dependencies). Currently still trying to figure out how to make an IPA file and upload it to my phone. I don't have a developer account, so Xcode is not working with me.

**Comment by arjes** [comment]
Hey,

Apple is kinda a pain like that. You can use nvm (node version manager) if
you need more than one node version.

I think your should still be able to use the simulator without a developer
account. If you still have problems you can also do dev against an Android
simulator and I'll test it against iOS.

**Comment by ualex73** [comment]
Just asking: Do you know if i can compile + load the app on an Android phone without a required subscription with google? In such case, i will order myself an Android ;-)

**Comment by arjes** [comment]
Yep with Android you only need an account to publish to the app store

**Comment by ualex73** [comment]
Ok, i got it compiled for Android and the emulator is working ;-) Only cannot test the scanner, maybe i will buy a cheap Android phone.

**Comment by arjes** [comment]
You should be able to use a webcam. It is clunky and grainy.but works.

Are you trying to make a change in specific?

**Comment by ualex73** [comment]
Yes, I am trying to add "Barcode Data Source" for the Albert Heijn (NL) - because my wife normally buys stuff there.

**Comment by arjes** [comment]
Awesome! Let me know if you need any help. They have a public api?
